Project_code/src/index.js

```diff
@@ -114,18 +118,17 @@ app.get("/register", (req, res) => {
   res.render("pages/login.ejs", { showSignUpPanel: true });
 });
 
-
 //Register post call
 app.post("/register", async (req, res) => {
   //hash the password using bcrypt library
-  const query = "INSERT INTO users (username, password, first_name, last_name, email, created_at) VALUES ($1, $2, $3, $4, $5, $6);";
-  const values = [req.body.username.trim(), hash, req.body.first_name.trim(), req.body.last_name.trim(), req.body.email.trim(), new Date()];
   let hash;
+  const query = "INSERT INTO users (username, password, first_name, last_name, email, created_at) VALUES ($1, $2, $3, $4, $5, $6);";
+  const values = [req.body.username, hash, req.body.firstName, req.body.lastName, req.body.email, new Date()];

```

Comment:
Not sure why you're changing to firstName and lastName throughout, this will create issues for other pull requests